.coverage.sh

```diff
@@ -8,23 +8,33 @@ mkdir -p $COVERAGE_DIR
 pushd $COVERAGE_DIR
 if [ -z "$KEEP" ]; then trap "popd; rm -rf $COVERAGE_DIR" EXIT; fi
 
+# copy over everything
 $(which cp) -r ../* .
 
+# prepare the environment
+eval `opam config env`
 opam install bisect_ppx ocveralls -y
 
+# run the tests with coverage enabled
 export BISECT_ENABLE=YES
 jbuilder runtest
 
-outs=$(find . | grep bisect.*.out)
-bisect-ppx-report -I $(dirname $outs[1]) -text report $outs
-bisect-ppx-report -I $(dirname $outs[1]) -summary-only -text summary $outs
-if [ -n "$HTML" ]; then bisect-ppx-report -I $(dirname $outs[1]) -html ../html-report $outs; fi
+# move all bisect files here
+find . -name 'bisect*.out' -exec mv {} . \;
+
+DIRECTORIES=`find . -type d`
```

Comment:
This is quite a nice pattern. The `printf` creates a string "-I xx -I yy -I zz" - one entry per directory.